src/datarecord.cpp

```diff
@@ -233,26 +230,22 @@ void datarecord::steady_bolus(odeproblem* prob, LSODA& solver) {
   }
   
   prob->rate_reset();
-  double N_SS = prob->ss_n;  
-  double CRIT_DIFF_SS = prob->ss_tol;
+  bool warn = !prob->ss_fixed;
+  int N_SS = prob->ss_n;  
+  int min_check = (N_SS < 10 || prob->ss_fixed) ? N_SS : 10;
+
```

Comment:
If I understand it correctly, this is essentially what `ss_fixed` is for: to control the number of dosing/ODE solutions, and with its `false` default we do maximum 10 doses. Does this number 10 cover majority of cases?

Reply:
- My thinking on this was to run 10 iterations up front before starting to check
- Some scenarios would get there before 10 but it could be all over the place
- Maybe it has limited usefulness